lib/chutney/TorNet.py

```diff
@@ -1757,7 +1757,7 @@ def isInExpectedDirInfoDocs(self):
         node_status = self.getNodeDirInfoStatus()
         if node_status:
             status_code, _, _, _ = node_status
-            return status_code == LocalDirectoryController.SUCCESS_CODE
+            return status_code == LocalNodeController.SUCCESS_CODE
```

Comment:
`LocalDirectoryController` is nowhere else in chutney. I think the coder meant `LocalNodeController`

Reply:
Weird, we must never run this code

Reply:
Chuntey should have unit tests for its functions.

Do we get to keep this change ??

Reply:
I agree that chutney should unit test each function, but it doesn't do that right now.

Yes, we should keep this change.